upload-to-release

```diff
@@ -3,6 +3,9 @@
 set -e
 set -o pipefail
 
+# Bail if this push doesn't include a tag
+[[ "$GITHUB_REF" != refs/tags/* ]] && echo "${GITHUB_REF} is not a tag, exiting..." && exit 78
```

Comment:
I'm not opposed to this change (more validation is always :sparkles:), but I'm not sure I understand the need. Would you mind elaborating?

Reply:
This prevents it uploading every single time you push vs just when you push a tag.

Reply:
This action triggering should be up to the person using it in their workflow - it can be triggered by a `push`, but also by a `release` event. So I think that checks like this end up being more restrictive than they are helpful.